session/channel.go

```diff
@@ -250,18 +254,21 @@ func (ch *Channel) SubChUpdates(notifier perun.ChUpdateNotifier) error {
 }
 
 // UnsubChUpdates implements chAPI.UnsubChUpdates.
-func (ch *Channel) UnsubChUpdates() error {
+func (ch *Channel) UnsubChUpdates() perun.APIErrorV2 {
 	ch.Debug("Received request: channel.UnsubChUpdates")
```

Comment:
Update the log message. See the above [comment](https://github.com/hyperledger-labs/perun-node/pull/157/files#r582712783).